app/views/api/v1/conversations/partials/_conversation.json.jbuilder

```diff
@@ -17,7 +17,7 @@ json.meta do
 end
 
 json.id conversation.display_id
-if conversation.messages.count.zero?
+if conversation.messages.first.blank?
   json.messages []
 elsif conversation.unread_incoming_messages.count.zero?
```

Comment:
we can avoid the count query here.  do first.present?

Reply:
I did `first.present?` earlier, then rubocop suggested to make it `first.blank?`Are the two different in their query execution?

Reply:
`first.blank?` should be good . lets do the same for  line:22 as well.  the count query seems overkill there as well

Reply:
We could do that but then `conversation.unread_incoming_messages.count` is fetched again on line 43, unconditionally. So I figured it better be fetched on 22 as well